internal/expr/bindtypes.go

```diff
@@ -93,8 +84,9 @@ func (b *bypass) String() string {
 
 // bindTypes returns the bypass part itself since it contains no references to
```

Comment:
Here and other places, you need to alter the comments around the fact that there are no returns, rather accumulations in the `typedExprBuilder`.

Reply:
Added them.